src/api/helpers/map-CIS-countries/map-CIS-country/can-apply-for-insurance-online/index.ts

```diff
@@ -9,16 +9,10 @@
  * - CILC
  * - Refer
  * - Unlisted
- * @param {Boolean} shortTermCover: Boolean value of shortTermCover.
- * @param {String} riskCategory: Country risk category.
+ * @param {Boolean} shortTermCover: Short term cover flag.
+ * @param {String} esraClassification: ESRA classification.
  * @returns {Boolean}
  */
-export const canApplyForInsuranceOnline = (shortTermCover: boolean, riskCategory?: string) => {
-  if (riskCategory && shortTermCover) {
-    return true;
-  }
-
-  return false;
-};
+const canApplyForInsuranceOnline = (shortTermCover: boolean, esraClassification: string | null) => shortTermCover && esraClassification;
```

Comment:
@Zainzzkk Had to revert this change, the one liner will not work as it can return string | null | boolean. It needs to be a boolean at all times.

Reply:
thats fine, will approve.  Otherwise we could have wrapped it in Boolean like `=> Boolean(shortTermCover && esraClassification);`